mig-blue-green/pipelines/apply.cloudbuild.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+
+- id: tf_apply
+  name: "gcr.io/cloud-builders/gcloud"
+  env:
+    - "PROJECT_ID=$PROJECT_ID"
+  script: |
+    #!/bin/bash
+    set -e
+    source /workspace/lib/bash_utils.sh
+    tf_install_in_cloud_build_step
+    tf_apply
```

Comment:
so we can control TF version exactly because Cloud Build image update policy is not explicitly defined

Reply:
The gcloud image doesn't need to be used. This breaks the entire concept of having Cloud Builders, please update.

Reply:
At the time of this writing Google Cloud Build official builder images repository (https://github.com/GoogleCloudPlatform/cloud-builders) does not have a Terraform builder. The Google Cloud Build community images repository has one (https://github.com/GoogleCloudPlatform/cloud-builders-community), but at a vesion 1.0.9. To take advantage of more recent Terraform 1.3.4, we decided to install it directly from the distribution instead of relying on pre-build images.